docs/about/how_can_i_try_mautic.rst

```diff
@@ -0,0 +1,8 @@
+How can I try Mautic?
+#####################
+
+Mautic is available to download for free [:xref:`Download Mautic`], allowing you to explore its full capabilities without restrictions when you host it yourself.
+
+For optimal performance, we recommend hosting Mautic on a Virtual Private Server (VPS) rather than shared hosting, particularly if you intend to use it in a production setting on lower end hosting which often have limited resources, which may impact Mautic's efficiency and performance.
```

Comment:
```suggestion
For optimal performance, it's recommended to host Mautic on a Virtual Private Server (VPS) rather than shared hosting, particularly if you intend to use it in a production setting on lower end hosting which often have limited resources, which may impact Mautic's efficiency and performance.
```